nilearn/maskers/nifti_labels_masker.py

```diff
@@ -112,6 +114,30 @@ class NiftiLabelsMasker(BaseMasker, _utils.CacheMixin):
 
         .. versionadded:: 0.9.2
 
+    region_ids_ : dict
+        A dictionary containing the region ids corresponding to each
+        column in region_signal. The region id corresponding to
+        region_signal[:,i] is region_ids_[i].
+        region_ids_['background'] is the background label.
+
+        .. versionadded:: 0.10.2.dev
```

Comment:
I'm wondering (though I can't check locally with due to the MKL doc dependency) if this version string is related to the Sphinx error we're seeing. I can only find one other instance of a `.dev` version string in the code base, though this is for the `deprecated` admonition and [in that case](https://github.com/nilearn/nilearn/blob/48aecfacd9f0411c50125658966d5a61f40ec2e3/nilearn/_utils/docs.py#L502) there's an included description afterwards.